bot/code_review_bot/revisions.py

```diff
@@ -206,17 +208,22 @@ def from_decision_task(task: dict, phabricator: PhabricatorAPI):
         """
         Build a revision from a Mozilla decision task (e.g. from Autoland or Mozilla-central)
         """
-        assert task["payload"]["env"]["GECKO_HEAD_REPOSITORY"] in (
+        # Load repositories
+        repository = task["payload"]["env"]["GECKO_HEAD_REPOSITORY"]
+        target_repository = task["payload"]["env"]["GECKO_BASE_REPOSITORY"]
+
+        assert repository in (
             REPO_AUTOLAND,
             REPO_MOZILLA_CENTRAL,
         ), "Decision task must be on autoland or mozilla-central"
 
-        # Load mercurial revision
+        # Load mercurial revisions
         mercurial_revision = task["payload"]["env"]["GECKO_HEAD_REV"]
```

Comment:
Same thing: please use `head_changeset` and `base_changeset`  to avoid confusion between mercurial revision and `Revision` instances, but also between base & head. Target is too broad